experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -9,9 +9,17 @@ <h1>Changes Overview</h1>
     <p>Experiment Slug: {{ experiment.slug }}</p>
     {% for changelog in experiment.get_changelogs %}
     <div>
-        <h2>{{ changelog.changed_on }}</h2>
-        <p><i class="fa-regular fa-pen-to-square"></i> : {{ changelog.message }}</p>  
-        <p><i class="fa-regular fa-user"></i> : {{ changelog.changed_by }}</p>
+        <h2>{{ changelog.date }}</h2>
+        {% for change in changelog.changes %}
+        <div>
+          {% if change.event == "GENERAL" %}
+            <p>Change Log: {{ change.changed_by }} changed the value of {{ change.field }} from {{ change.old_value }} to {{ change.new_value }}</p>  
+          {% elif change.event == "CREATION" %}
+            <p>Change Log: {{ change.changed_by }} created this experiment</p>  
+          {% endif %}
+          <p>Changed at: {{ change.timestamp }}</p>
+        </div>
```

Comment:
you can avoid putting logic here and can put a key like `message` and build message there because later we will be having more events like review etc etc, so it doesn't make sense to have all logic processing here

Reply:
I noticed we already use `message` so you can use like `event_message`

Reply:
So the reason I made them into separate fields is to make the implementation more flexible, like different styling for the values, the user or the timestamp  so that they are more distinguished